contracts/libraries/SecondsUntilNextSwap.sol

```diff
@@ -26,10 +26,10 @@ library SecondsUntilNextSwap {
     uint256 _smallerIntervalBlocking;
     while (_activeIntervals >= _mask && _mask > 0) {
       if (_activeIntervals & _mask == _mask) {
-        IDCAHub.SwapData memory _swapDataMem = _hub.swapData(_tokenA, _tokenB, _mask);
```

Comment:
Should I build another version for the legacy hubs? I don't think anyone was using this 😅